src/Symfony/Component/EventDispatcher/EventDispatcher.php

```diff
@@ -80,7 +80,7 @@ public function getListeners($eventName = null)
      */
     public function hasListeners($eventName = null)
     {
-        return (bool) count($this->getListeners($eventName));
+        return (bool) $this->getListeners($eventName);
```

Comment:
Personally I would prefer to see `return 0 !== count($this->getListeners($eventName));` If for some reason, it's decided to return an object instead, it will always return true, as the object is casted to a boolean, and empty objects are also true.

https://3v4l.org/O8WGj